UnchainedLauncherGUI/src/ViewModels/ModViewModel.cs

```diff
@@ -45,14 +47,14 @@ public bool HasError {
             }
         }
 
-        public Option<Either<DisableModFailure, EnableModFailure>> ErrorState { get; set; }
+        public Either<DisableModFailure, EnableModFailure>? ErrorState { get; set; }
 
 
-        public Option<Release> EnabledRelease { get; set; }
+        public Release? EnabledRelease { get; set; }
```

Comment:
In general I prefer Option over nullable, because Option gives quite a few niceties which helps with composing computations out of optional values while doing a better job at preserving type information.  However for WPF view bindings, Option acts funny, so these use nullable.